desktop/main.dev.ts

```diff
@@ -61,7 +61,7 @@ let browserView: BrowserView;
 let tray: Tray;
 let nodeManager: NodeManager;
 let notificationManager: NotificationManager;
-let isDarkMode: boolean = nativeTheme.shouldUseDarkColors;
```

Comment:
So now the App will not take into account the theme of the OS.
The `nativeTheme.shouldUseDarkColors` should be a default.